src/app/templates/manifest-only/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "<%= projectInternalName %>",
+  "name": "office-addin-manifest",
```

Comment:
Is this change OK because we use office-addin-manifest to customize the name?

Reply:
reverting this change